courses/models.py

```diff
@@ -19,6 +20,24 @@ class Program(models.Model):
     def __str__(self):
         return self.title
 
+    def get_course_price(self):
+        """
+        Returns a decimal course price attached to this program.
+
+        Note: This implementation of retrieving a course price is a naive lookup that assumes
+        all course runs and courses will be the same price for the foreseeable future.
+        Therefore we can just take the price from any currently enroll-able course run.
+        """
+        from ecommerce.models import CoursePrice
```

Comment:
This would result in a circular import since `ecommerce.models` imports `courses.models`